### DISCUSSION TITLE
PMV AND PPD CALCULATION ACCORDING ASHRAE 55

### DISCUSSION THREAD

**Discussion by mm834** [CenterForTheBuiltEnvironment/pythermalcomfort#81] [Q&A]
Why using the example code and changing the values of tdb, tr, vr, met and clo many times I get pmv=nan, ppd=nan ??

**Comment by FedericoTartarini** [comment]
Please read the official documentation. You are getting nan since you are using inputs outside the standard applicability limits. You can turn off this feature by setting the input variable 'limit_inputs' to false but I would not suggest doing that.

**Reply by gtmlou** [reply]
Many thanks for your early response. I had already read the official documentation:
The ASHRAE 55 2020 limits are 10 < tdb [°C] < 40, 10 < tr [°C] < 40, 0 < vr [m/s] < 2, 1 < met [met] < 4, and 0 < clo [clo] < 1.5. The ISO 7730 2005 limits are 10 < tdb [°C] < 30, 10 < tr [°C] < 40, 0 < vr [m/s] < 1, 0.8 < met [met] < 4, 0 < clo [clo] < 2
and although I tried values in the above ranges, many times  I was getting 'nan'. 
Why you don't suggest setting the input variable 'limit_inputs' to false? 
Many thanks

**Reply by FedericoTartarini** [reply]
Could you please let me know for which combinations of inputs within those ranges you get nan?

I would not suggest removing this filter since results beyond the model applicability limits are most likely to be inaccurate.

**Comment by mm834** [comment]
Many thanks for your early response. I had already read the official documentation:
The ASHRAE 55 2020 limits are 10 < tdb [°C] < 40, 10 < tr [°C] < 40, 0 < vr [m/s] < 2, 1 < met [met] < 4, and 0 < clo [clo] < 1.5. The ISO 7730 2005 limits are 10 < tdb [°C] < 30, 10 < tr [°C] < 40, 0 < vr [m/s] < 1, 0.8 < met [met] < 4, 0 < clo [clo] < 2
And although I tried values in the above ranges, I was getting 'nan' many times. Are there any other limit ranges that the code considers?
Why don't you suggest setting the input variable 'limit_inputs' to false?
Many thanks
________________________________

**Comment by mm834** [comment]
For example, with inputs: tdb=22, tr=17, rh=45.8, v=0.1, met=1.2, clo=0.35 and without the filter I get ''nan''

whereas with the filter I get pmv=-2.24  and ppd=86.3

Thanks
________________________________

**Comment by FedericoTartarini** [comment]
Yes, this is expected since the ISO Standard specify that the PMV model should only be used when PMV < |2|.
